### PR TITLE
[fix] setuptools: include ALL files

### PR DESCRIPTION
**Pull request by simahawk**
ping @sbidoul :)

### CODE REVIEW

File: setup/cms_form/setup.py [hunk 1]
```diff
@@ -2,5 +2,7 @@
 
 setuptools.setup(
     setup_requires=['setuptools-odoo'],
-    odoo_addon=True,
+    odoo_addon={'odoo_version_override': '10.0'},
```
Comment (sbidoul) [inline setup/cms_form/setup.py:5]: this is automatic if the version in `__manifest__.py` starts with 10.0

File: setup/cms_form/setup.py [hunk 2]
```diff
@@ -2,5 +2,7 @@
...
+    include_package_data=True,
```
Comment (sbidoul) [inline setup/cms_form/setup.py:6]: `include_package_data` is implicit with `odoo_addons=True`

File: setup/cms_form/setup.py [hunk 3]
```diff
@@ -2,5 +2,7 @@
...
+    package_data={'': ['*', ]}
```
Comment (sbidoul) [inline setup/cms_form/setup.py:7]: the setuptools documentation is not very clear (to me), but I understand `package_data` is not necessary if you have a MANIFEST.in

File: setup/cms_form/MANIFEST.in [hunk 4]
```diff
@@ -0,0 +1 @@
+global-include *.md *.rst *.py *.xml *.csv *.tmpl
```
Comment (sbidoul) [inline setup/cms_form/MANIFEST.in:1]: What about .js, .html, .css? include static/**?

File: setup/cms_form/setup.py (hunk 1 above)
Comment (sbidoul) [reply setup/cms_form/setup.py:5]: so in this case `odoo_addons=True` is equivalent

File: setup/cms_form/setup.py (hunk 2 above)
Comment (simahawk) [reply setup/cms_form/setup.py:6]: wouldn't be better if we put this explicitly? 
Easy to check what's in the setup and if Odoo will ever move to real py pkgs then is going to be already there. No?

File: setup/cms_form/setup.py (hunk 3 above)
Comment (simahawk) [reply setup/cms_form/setup.py:7]: no very clear to me neither ;) but yes, should be like this. To be honest: I was trying both and then I left some dirt behind :P 
I'll clean up

**Pull request closed without merging**